css/syntaxes.json

```diff
@@ -750,7 +753,7 @@
     "syntax": "running | paused"
   },
   "single-animation-timeline": {
-    "syntax": "auto | none | <timeline-name> | scroll(<axis>? <scroller>?)"
+    "syntax": "auto | none | <custom-ident> | <scroll()> | <view()>"
```

Comment:
[Spec says `dashed-ident`](https://drafts.csswg.org/css-animations-2/#typedef-single-animation-timeline) instead of custom, or is this what Bramus was clarifying earlier?

Reply:
Oops, I missed this one. Updated in next commit! ;-)